torch_geometric/nn/conv/disen_conv.py

```diff
@@ -0,0 +1,155 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+from torch_sparse import SparseTensor, matmul
+
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.nn.dense.linear import Linear
+
+
+class DisenConv(MessagePassing):
+    r"""The graph convolutional operator from the `"Disentangled graph
```

Comment:
Can we add more information about the operator here (similar to other GNN layers)?

Reply:
Yes. Once we reach a consensus on the code.